tests/pos/i6255.scala

```diff
@@ -0,0 +1,7 @@
+class Foo {
+  def foo(x: quoted.Expr[Int]) given scala.tasty.Reflection: Unit = x match {
+    case '{ 1 } =>
+    case '{ 2 } =>
+    case _ =>
```

Comment:
Shouldn't this be a pattern matching test ? Also do we get a warning if `case _` is missing ?

Reply:
Good catch, thanks. If we skip the check as it's done in this PR, we will not get the warning. I agree it will be helpful to issue an issue here. I'll update the PR to support that.